apps/dashboard/src/components/primitives/field-editor/variable-popover/components/draggable-transformer.tsx

```diff
@@ -0,0 +1,131 @@
+import { Button } from '@/components/primitives/button';
+import { Input } from '@/components/primitives/input';
+import { GripVertical } from 'lucide-react';
+import { motion } from 'motion/react';
+import { forwardRef } from 'react';
+import { RiCloseLine } from 'react-icons/ri';
+import { TRANSFORMERS } from '../constants';
+import { TransformerWithParam } from '../types';
+
+type DraggableTransformerProps = {
+  transformer: TransformerWithParam;
+  index: number;
+  isLast: boolean;
+  draggingItem: number | null;
+  dragOverIndex: number | null;
+  onDragStart: (index: number) => void;
+  onDragEnd: () => void;
+  onDrag: (e: any, info: any) => void;
+  onRemove: (value: string) => void;
+  onParamChange: (index: number, params: string[]) => void;
+};
+
+export const DraggableTransformer = forwardRef<HTMLDivElement, DraggableTransformerProps>(
```

Comment:
This component deserves being a top level primitive called Sortable, used from the variable popover, to drag and sort Liquid filters.

Reply:
I have left it out intentionally for the time being, I'm not sure entirely where and how this is going to be used in other places, and would prefer to extract it when needed as it's not a primitive on our design system yet.